apps/www/data/products/vector/pageData.tsx

```diff
@@ -12,14 +12,15 @@ export default {
     title: 'Supabase Vector',
     h1: (
       <span key={'vector-h1'} className="heading-gradient">
-        The open source Vector Toolkit <br />
-        for Postgres
+        The best vector database  <br />
```

Comment:
I think I like "The open source Vector Toolkit for Postgres" for the main header. It communicates the "what is this?" quite well